app/code/Magento/Backup/Controller/Adminhtml/Index/Create.php

```diff
@@ -55,20 +55,20 @@ public function execute()
             $this->_coreRegistry->register('backup_manager', $backupManager);
 
             if ($this->getRequest()->getParam('maintenance_mode')) {
-                if (!$this->maintenanceMode->set(true)) {
-                    $response->setError(
-                        __(
-                            'You need more permissions to activate maintenance mode right now.'
-                        ) . ' ' . __(
-                            'To create the backup, please deselect '
-                            . '"Put store into maintenance mode" or update your permissions.'
-                        )
-                    );
-                    $backupManager->setErrorMessage(
-                        __("Something went wrong while putting your store into maintenance mode.")
-                    );
-                    return $this->getResponse()->representJson($response->toJson());
-                }
+                $this->maintenanceMode->set(true);
+            } else {
+                $response->setError(
```

Comment:
Displaying insufficient permissions error in case `maintenance_mode` parameter is false is logically incorrect